hikari/guilds.py

```diff
@@ -2501,7 +2501,7 @@ def make_banner_url(self, *, ext: str = "png", size: int = 4096) -> typing.Optio
         ----------
         ext : builtins.str
             The extension to use for this URL, defaults to `png`.
-            Supports `png`, `jpeg`, `jpg` and `webp`.
```

Comment:
should it not just be fixed to also allow "jpg" since that'd a better match the cdn's behaviour and b just be less strict ("jpg" and "jpeg" are just straight out used interchangeably and arbitrarily only allowing one could end up being annoying)

Reply:
Discord don't document as allowing `jpg`, but I haven't really tested this

Reply:
https://discord.com/developers/docs/reference#image-formatting-image-formats

Reply:
Completely missed that :sweat_smile: